metadata.json

```
@@ -103,7 +103,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">=4.10.0 <7.0.0"
+      "version_requirement": ">=5.0.0 <7.0.0"
```

I think the minimum version we actually support is higher than this.

```suggestion
      "version_requirement": ">= 5.5.8 < 7.0.0"
```